.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "node"
```

Comment:
Apparently unless I do this, the version of node that is run blows up on `const something = ...` in one of the dependencies.  I learned about the fix here https://codepunk.io/travis-ci-uncaught-syntaxerror-use-of-const-in-strict-mode/